danil.andryushin/task-2-1/internal/conditioner/depatment_temperature.go

```diff
@@ -0,0 +1,71 @@
+package conditioner
+
+import (
+	"errors"
+	"fmt"
+	"io"
+)
+
+var ErrInvalidEmployeesCount = errors.New("invalid employees count")
+
+func CalcDepartmentTemperature(reader io.Reader, writer io.Writer) (int, error) {
+	var (
+		nEmployees  uint
+		wish        = Wish{15, 30}
+		temperature int
+	)
+
+	_, err := fmt.Fscan(reader, &nEmployees)
+	if err != nil {
+		return 0, ErrInvalidEmployeesCount
+	}
+
+	for range nEmployees {
+		temperature, err = fulfilWish(&wish, reader)
+		if err != nil {
+			return 0, err
+		}
+
+		if writer != nil {
+			_, err = fmt.Fprintln(writer, temperature)
+			if err != nil {
+				return 0, fmt.Errorf("invalid output stream: %w", err)
+			}
+		}
+	}
+
+	return temperature, nil
+}
+
+var (
+	ErrReadWish        = errors.New("failed to read employee wish")
```

Comment:
да и так в целом понятно, что `failed`, поэтому в ошибке такие слова принято не писать)
```suggestion
	ErrReadWish        = errors.New("read employee wish")
```